src/mmw/apps/modeling/tasks.py

```diff
@@ -19,6 +19,51 @@
 CM_PER_INCH = 2.54
 
 
+@shared_task
+def polygon_to_id(json_polygon):
+    polygon = json.loads(json_polygon)
+    return {
+        'pixel_width': aoi_resolution(polygon),
+        'sjs_job_id': histogram_start([json_polygon])
+    }
+
+
+@shared_task
+def polygons_to_id(polygons):
+    json_polygons = [json.dumps(p) for p in polygons]
+    return {
+        'pixel_width': None,
+        'sjs_job_id': histogram_start(json_polygons)
+    }
+
+
+@shared_task(bind=True, default_retry_delay=2)
+def id_to_histogram(self, incoming):
```

Comment:
Poll for the result every two seconds.  Unfortunately, I do not currently know how to put a limit on the number of times this occurs.